compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/CacheConfigContext.java

```diff
@@ -0,0 +1,33 @@
+package io.ballerina.stdlib.graphql.compiler;
+
+public class CacheConfigContext {
+    private Boolean enabled;
+    private int maxSize;
+
+    public CacheConfigContext(Boolean enabled, int maxSize) {
```

Comment:
Do we need these methods to be public?

Reply:
yes

Reply:
Who is accessing them from the outside? As it seems these can be package private.

Reply:
It used inside the service validator.